File: backend/Domain/Entities/structure_tag_application.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Domain.Entities
{
    public class structure_tag_application
    {
        public int id { get; set; }
		public int structure_tag_id { get; set; }
		public string structure_tag_name { get; set; }
		public string structure_tag_code { get; set; }
        public int application_id { get; set; }
		public DateTime? created_at { get; set; }
		public DateTime? updated_at { get; set; }
		public int? created_by { get; set; }
		public int? updated_by { get; set; }
		public int structure_id { get; set; }
		
    }
}

File: frontend/src/constants/structure_tag_application.ts
import { Dayjs } from "dayjs";

export type structure_tag_application = {
  
  id: number;
  structure_tag_id: number;
  application_id: number;
  created_at: Dayjs;
  updated_at: Dayjs;
  created_by: number;
  updated_by: number;
  structure_id: number;
};
